src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/ContributorTeamRepository.cs

```diff
@@ -84,7 +87,11 @@ public IQueryable<ContributorTeam> FindWhere(System.Linq.Expressions.Expression<
         /// <param name="newEntity">The contributor team for adding.</param>
         public void Add(ContributorTeam newEntity)
         {
-            throw new NotImplementedException();
+            var newUser = new ContributorTeamEntity();
```

Comment:
What is the reason for this change?

Reply:
implementing this method instead of throwing exception. Revert changes?

Reply:
ok